imap_processing/cdf/config/imap_ultra_l1b_variable_attrs.yaml

```diff
@@ -215,6 +215,20 @@ elevation:
   LABLAXIS: elevation
   UNITS: radians
 
+phi:
+  <<: *default_float32
+  CATDESC: Ultra instrument frame event phi.
```

Comment:
It might help to add what coordinate this refers to, or put a min/max to indicate what you expect for phi. Phi is just a generic angle otherwise.